packages/core/src/hooks/useSendTransaction.ts

```diff
@@ -0,0 +1,83 @@
+import { Call } from "starknet";
+import { Call as RequestCall } from "starknet-types";
+import {
+  RequestArgs,
+  RequestResult,
+  UseWalletRequestProps,
+  UseWalletRequestResult,
+  useWalletRequest,
+} from "./useWalletRequest";
+
+type UseSendTransactionArgs = {
+  /** List of smart contract calls to execute. */
+  calls: Call[];
```

Comment:
```suggestion
  calls?: Call[];
```

Like other APIs, it should be allowed to be `undefined` and in that case it won't send the transaction.